src/lib/containers/dataaccess/AccessSubmissionDashboard.tsx

```diff
@@ -0,0 +1,134 @@
+
+export type DataAccessSubmissionDashboardProps = {}
+export const DataAccessSubmissionDashboard: React.FunctionComponent<
+  DataAccessSubmissionDashboardProps
+> = () => {
+  const [accessRequirementId, setAccessRequirementId] = useState<
+    string | undefined
+  >()
+  const [accessorId, setAccessorId] = useState<string | undefined>()
+  const [reviewerId, setReviewerId] = useState<string | undefined>()
+  const [tableProps, setTableProps] =
+    useState<AccessRequestSubmissionTableProps>({
+      accessRequirementId,
+      accessorId,
+      reviewerId,
+      showRequesters: true,
+    })
+
+  const location = useLocation()
+  const history = useHistory()
+  const INPUT_CHANGE_DEBOUNCE_DELAY_MS = 500
+
+  useEffect(() => {
+    const initializeFromSearchParam = () => {
+      const initialParams = new URLSearchParams(location.search)
+      setAccessRequirementId(
+        initialParams.get('accessRequirementId') ?? undefined,
+      )
+      setAccessorId(initialParams.get('accessorId') ?? undefined)
+      setReviewerId(initialParams.get('reviewerId') ?? undefined)
+    }
+    initializeFromSearchParam()
+  }, [location.search])
+
+  const onRequesterChange = useCallback((selected: string | null) => {
+    if (selected) {
+      setAccessorId(selected)
+    } else {
+      setAccessorId(undefined)
+    }
+  }, [])
+
+  const onReviewerChange = useCallback((selected: string | null) => {
+    if (selected) {
+      setReviewerId(selected)
+    } else {
+      setReviewerId(undefined)
+    }
+  }, [])
+
+  useDebouncedEffect(
+    () => {
+      const updateQueryParams = (
+        accessRequirementId: string | undefined,
+        accessorId: string | undefined,
+        reviewerId: string | undefined,
+      ) => {
+        const params = new URLSearchParams(
+          omitBy(
+            {
+              accessRequirementId,
+              accessorId,
+              reviewerId,
+            },
+            item => item === undefined || item === '',
+          ) as Record<string, string>,
+        )
+        history.replace({
+          pathname: location.pathname,
+          search: params.toString(),
+        })
+      }
+
+      updateQueryParams(accessRequirementId, accessorId, reviewerId)
+      setTableProps({
+        accessRequirementId,
+        accessorId,
+        reviewerId,
+        showRequesters: true,
+      })
+    },
+    [accessRequirementId, accessorId, reviewerId, history, location],
+    INPUT_CHANGE_DEBOUNCE_DELAY_MS,
+  )
+
+  return (
+    <div className="AccessSubmissionDashboard bootstrap-4-backport">
+      <div className="InputPanel SubmissionGrid">
+        <div>
+          <FormLabel htmlFor="arId-filter">
+            Filter by Access Requirement Name
+          </FormLabel>
+          <AccessRequirementSearchBox
+            onChange={setAccessRequirementId}
+            placeholder="Search for an Access Requirement Name "
+          />
```

Comment:
Can you add a prop `htmlId` to AccessRequirementSearchBox as in UserSearchBoxV2?

Then use the prop here to map the label to the input
```suggestion
          <FormLabel htmlFor="arId-filter">
            Filter by Access Requirement Name
          </FormLabel>
          <AccessRequirementSearchBox
            htmlId="arId-filter"
            onChange={setAccessRequirementId}
            placeholder="Search for an Access Requirement Name "
          />
```